nvflare/app_common/storages/filesystem_storage.py

```diff
@@ -12,44 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import inspect
 import os
 import pickle
 import shutil
-from functools import wraps
 from pathlib import Path
 from typing import ByteString, List, Tuple
 
 from nvflare.apis.storage import StorageSpec
+from nvflare.apis.utils.format_check import validate_class_methods_args
 
+URI_ROOT = os.path.abspath(os.sep)
```

Comment:
suggest `URI_ROOT= "/"`

I think os.sep does not always equal to the root directory of the system.

Reply:
`os.path.abspath` called on `os.sep` does give the root directory of the system regardless of the OS

Reply:
Got it, sounds good.